app/controllers/admin/organization_contacts_controller.rb

```diff
@@ -47,11 +48,15 @@ def create
     end
 
     def destroy
```

Comment:
I have largely the same aloud thoughts about slimming down this method I did as the `ContactsController`.